app/js/script.js

```diff
@@ -211,9 +198,40 @@ const showCaseSuccess = () => {
   caseError.textContent = '';
 };
 
+// Store and retrieve user information
+
+// Store user info
+const storeUserInfo = () => {
+  const theUser = {};
+  const userName = document.querySelector('#name').value;
+  const userEmail = document.querySelector('#email').value;
+  const userMessage = document.querySelector('#message').value;
+
+  theUser.name = userName;
+  theUser.email = userEmail;
+  theUser.message = userMessage;
+
+  // Saving to local storage
+  localStorage.setItem('theUser', JSON.stringify(theUser));
+};
```

Comment:
- [ ] Please notice that we are not calling this function anywhere and because of that the data is not saved  into localstorage as it should, we need to store the user-entered data from our form fields to localstorage , you can use a 'change' eventlistener to listen for the change event in your form field and then call the` storeUserInfo() `function, so that on each change it will listen for the user input and save it to localstorage, something like this :point_down: 
```
const f = document.querySelectorAll('.footer__form')

f.forEach(i => {
  i.addEventListener('change', storeUserInfo)
})
```

Reply:
Thanks for the suggestion, @PinkMoon25 it looks effective. 

When I tested the code, it stored the data. When I leave the formspree success page and go back to the portfolio page, I always find the form pre-filled with data saved on local storage. 

The form is always pre-filled when I reload the page as well. Is there a need for me to change this code?